boards/arm/mps2_an385/mps2_an385.yaml

```diff
@@ -7,5 +7,6 @@ toolchain:
   - zephyr
   - gnuarmemb
   - xtools
+  - xtools
```

Comment:
@galak: There was already xtools for mps2_an385.yaml, so now there're two in row.